apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -290,6 +291,22 @@ public int Timeout
             set => SetInt32WithValidation(nameof(Timeout), 0, int.MaxValue, value);
         }
 
+        /// <summary>
+        /// Request CommitStats for all read/write transactions throughout the
+        /// lifetime of the connection and log these.
+        /// </summary>
+        /// <remarks>
+        /// CommitStats that are returned for a transaction are logged using the
```

Comment:
```suggestion
        /// Commit statistics that are returned for a transaction are logged using the
```
(And everywhere else)